tests/functions/test_select.py

```diff
@@ -41,6 +82,13 @@ def test_select_rows_only(dataframe):
     assert_frame_equal(actual, expected)
 
 
+def test_select_rows_only_args(dataframe):
+    """Test output for rows only"""
+    actual = dataframe.select({"B": "two"}, axis="index")
+    expected = dataframe.loc(axis=0)[(slice(None), "two")]
+    assert_frame_equal(actual, expected)
+
+
 def test_select_rows_scalar_(dataframe):
     """Test output for rows only"""
     actual = dataframe.select(rows="bar")
```

Comment:
Should we also change the tests here to use `index=`? instead of `rows=`, otherwise I'm expecting we'll get a bunch of warnings in our test output.